bundles/admin/admin-layereditor/instance.js

```diff
@@ -56,16 +65,49 @@ Oskari.clazz.defineES('Oskari.admin.admin-layereditor.instance',
             // update all layers at once since we suppressed individual events
             const event = Oskari.eventBuilder('MapLayerEvent')(null, 'tool');
             this.sandbox.notifyAll(event);
+        }
 
+        /**
+         * Adds admin tools to layer list
+         */
+        _setupAdminTooling () {
             // add layerlist tool for adding new layers
             const toolingService = this.sandbox.getService('Oskari.mapframework.service.LayerListToolingService');
             if (toolingService) {
```

Comment:
Switch this to:
```
if (!toolingService) {
    return;
}
```
To get rid of the massive if block

Reply:
Done.